docs/jq.js

```diff
@@ -0,0 +1,6570 @@
+// Copyright 2010 The Emscripten Authors.  All rights reserved.
```

Comment:
Because jq.o, jq.wasm and jq.js are all outputs of the build process, we can keep those files off the code repo.

Reply:
This is also for GitHub pages, because most visitors would like to see the result deployed live :)

Reply:
Yeah I'd rather have it hosted at jqkungfu.com and avoid the code duplication and build outputs in the repo

Reply:
okey doke, I'll remove them all.